flow/skia_gpu_object_unittests.cc

```diff
@@ -137,23 +137,27 @@ TEST_F(SkiaGpuObjectTest, ObjectResetTwice) {
 }
 
 TEST_F(SkiaGpuObjectTest, UnrefResourceContextInTaskRunnerThread) {
-  std::shared_ptr<fml::AutoResetWaitableEvent> latch =
+  std::shared_ptr<fml::AutoResetWaitableEvent> latch1 =
```

Comment:
nit: consider more descriptive names, or just reuse the latch - once it's signaled it should be safe to reuse, we were just missing the second flush of the task runner.